lib/jira/resource/agile.rb

```diff
@@ -31,6 +31,18 @@ def self.get_sprint_issues(client, sprint_id, options = {})
         parse_json(response.body)
       end
 
+      def self.get_projects_full(client, board_id, options = {})
+        response = client.get("/rest/agile/1.0/board/#{board_id}/project/full")
+        parse_json(response.body)
+      end
+
+      def self.get_projects(client, board_id, options = {})
+        options[:maxResults] ||= 100
+        options[:startAt] ||= 0
+        response = client.get("/rest/agile/1.0/board/#{board_id}/project?maxResults=#{options[:maxResults]}&startAt=#{options[:startAt]}")
```

Comment:
You could also use `hash_to_query_string` https://github.com/exadeci/jira-ruby/blob/master/lib/jira/base.rb#L501
I used it here: https://github.com/exadeci/jira-ruby/blob/c6ef13f25855c1a762dc7b203710d14c96dc5219/lib/jira/resource/createmeta.rb#L35